geodiff/src/drivers/sqliteutils.cpp

```diff
@@ -47,7 +50,10 @@ void Sqlite3Db::create( const std::string &filename )
   int rc = sqlite3_open_v2( filename.c_str(), &mDb, SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE, nullptr );
   if ( rc )
   {
-    throw GeoDiffException( "Unable to create " + filename + " as sqlite3 database" );
+    std::string errMsg;
+    if ( mDb )
+      errMsg = sqlite3_errmsg( mDb );
+    throw GeoDiffException( "Unable to create " + filename + " as sqlite3 database (" + errMsg + ")" );
```

Comment:
suggestion: 
if `mDB==0` then the error is for example `Unable to open myfile.gpkg as sqlite3 database ()`, maybe add unknown error or just omit bracket?